testdata/dnn/.gitignore

```diff
@@ -1,4 +1,3 @@
-torch/*.txt
```

Comment:
This script will generate test data to the torch/ directory. But to invoke it one needs torch to be installed. I just uploaded all this generated data to make it possible test torch part of dnn without installing torch.